### DISCUSSION TITLE
Future maintenance

### DISCUSSION THREAD

**Discussion by ihavespoons** [gophish/gophish#3233] [General]
Hi @jordan-wright and any other maintainers,

I just wanted to see if there is any intent to maintain gophish going forward?

If not I am keen to put my hat in the ring to start contributing and maintaining as I think there isn't really another tool quite like this in the ecosystem?

Thanks,

**Comment by AT-StephenDetomasi** [comment]
Just been researching alternatives and it seems to be in a bit of a league of it's own. Would be such a shame for it to be abandoned
